src/main.py

```diff
@@ -1228,13 +1220,13 @@ def _return_to_cluster(self, new_primary, role, is_dead=False):
         """
         Return to cluster (try stupid method, if it fails we try rewind)
         """
-        logging.info('Starting returning to cluster.')
+        logging.info('ACTION. Starting return to cluster. New primary: {}'.format(new_primary))
```

Comment:
I think it is not an `ACTION`.
ACTION is a simple atomic operation, like switching some setting, service, running command, but not this big complex deed.